helpers/contains.js

```diff
@@ -12,8 +10,14 @@ const _ = require('lodash');
 const factory = () => {
     return function() {
         var args = Array.prototype.slice.call(arguments, 0, -1),
```

Comment:
We need to avoid slicing the arguments array.

Reply:
ok.  will change it here too.